File: onem2m-application/nodejs/onem2m-get-latest-cin.js
// Version 1.0*

var request = require('request');

var sensorToMonitor = "TiltSensor";
var originator = "Cae-" + sensorToMonitor;

var options = {
	uri: "http://127.0.0.1:8080/server/" + sensorToMonitor + "/DATA/la",
	method: "GET",
	headers: {
		"X-M2M-Origin": originator ,
		"Content-Type": "application/json"
	}
};

console.log("Sending request >>> ");
console.log(options.method + " " + options.uri);

request(options, function (err, resp, body) {
	console.log("\n\n<<< Response received ! ");
	if(err) {
		console.log("Error = " + err);
	} else {
		console.log("Response Status Code = " + resp.statusCode);
		console.log("Response Body = " + body);

		var obj = JSON.parse(body);
		console.log("\n\nEffective content of CIN = " + obj["m2m:cin"].con);
	}
});	

File: onem2m-application/python/onem2m-get-latest-cin.py
# Version 1.1

import requests
import json

sensorDataContainer = "TiltSensor/DATA"

print("Sending request >>> ")
response = requests.get('http://127.0.0.1:8080/server/' + sensorDataContainer + '/la', 
                        headers={'Accept': 'application/json'})

print("<<< Response received ! ")

if response.status_code != 200:
    print("Error = ", response.text)
else:
    cin = json.loads(response.content)
    print("Effective content of CIN = ", cin['m2m:cin']['con'])